.travis.yml

```diff
@@ -6,7 +6,7 @@ matrix:
   include:
     - rvm: 2.3.4
       env: "RAILS_VERSION=4.2.8"
-    - env: "RSOLR_VERSION=2.0.1"
+    - env: "RAILS_VERSION=5.1.1"
```

Comment:
Removing the RSOLR_VERSION makes this build only have a single job.  Prior to removing the RSOLR_VERSION it was building on rails 5.1.1 (latest) and 4.2.8.